st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -212,8 +259,40 @@ def get_system_information():
     return system_information
 
 
+def format_output_filename(cmd):
+    """"
+    Format the file name such as removing white spaces and special characters.
+    :param cmd: shell command
+    :return: formatted output file name
+    :rtype: ``str``
+    """
+    for char in cmd:
+        if char in ' !@#$%^&*()[]{};:,./<>?\|`~=+"':
+            cmd = cmd.replace(char, "")
+    return cmd
+
+
+def get_commands_output(config_yaml):
+    """"
+    Get output of the required shell command and redirect the output to a file.
```

Comment:
I like the "commands" section idea. 

I could see it come handy in scenarios where you want to include some additional information :+1: